docs/guides/data_audit.md

```
@@ -0,0 +1,202 @@
+      ],
+      "simhash_distinct": 1180,
+      "near_duplicate_pairs": 60,
+      "pii_counts": {"email": 18, "phone": 4}
+    }
+  }
+}
+```
+
+### Cross-split overlap
+
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)